content/blog/announcing-hpe-swarm-learning-2-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing HPE Swarm Learning 2.0.0
+date: 2023-05-11T04:47:46.996Z
+featuredBlog: true
+author: HPE Swarm Learning Team
+authorimage: /img/Avatar1.svg
+disable: false
+---
```

Comment:
Please add a tag:
swarm-learning

This will allow to automatically tie the blog post with the Swarm Learning landing page on HPE Developer Community Portal.

Reply:
Created tag.